habitat_baselines/agents/slam_agents.py

```diff
@@ -1,39 +1,38 @@
+# https://sumit-ghosh.com/articles/python-download-progress-bar/
```

Comment:
Is this getting used somewhere?

Also I just noticed that this file is missing the facebook header, I can add it in another PR.

Reply:
Looks like that comment is referencing this function: https://github.com/facebookresearch/habitat-api/blob/master/habitat_baselines/agents/slam_agents.py#L39

Moved it to there